packages/server/src/helpers/session-middleware.js

```diff
@@ -25,13 +25,12 @@ export default () => {
   })
 
   return session({
-    store,
-    key: 'aragonCourtSessionID',
+    // store,   // I disabled this for now because it doesn't work and I will be migrating to Objection in the next PR
```

Comment:
But front end guys need something functional right now so we can have it working like this (as it was working in my first implementation already)

Reply:
Can't we simply store a session for the users then? It would be simply doing [this](https://github.com/aragonone/court-backend/blob/development/packages/server/src/controllers/AdminsController.js#L20-L21) I think

Reply:
Oh if it's so simple I will fix it.

Reply:
Hmm in order to make this work your model forces me to use users table but I don't want to touch it right now while we are still actively developing. I would prefer to merge this in memory solution now while I figure out the storage stuff which will still take a day or so.

Reply:
added your fix, thanks!